com.unity.ml-agents/Tests/Editor/Integrations/Match3/Match3ActuatorTests.cs

```diff
@@ -194,5 +194,14 @@ public void TestMasking(bool fullBoard)
             // And they should add up to all the potential moves
             Assert.AreEqual(validIndices.Count + masks.HashSets[0].Count, board.NumMoves());
         }
+
+        [Test]
+        public void TestNoBoardReturnsEmptyActuators()
+        {
+            var gameObj = new GameObject("board");
+            var actuatorComponent = gameObj.AddComponent<Match3ActuatorComponent>();
+            var actuators = actuatorComponent.CreateActuators();
+            Assert.AreEqual(0, actuators.Length);
```

Comment:
might be good to also check it doesn't throw exceptions

Reply:
If it did throw exceptions, the test would fail.

Reply:
Oh ok then it's good